src/pass/vectorize_loop.cc

```diff
@@ -300,7 +300,6 @@ class Vectorizer : public IRMutator {
     CHECK(!op->condition.type().is_vector());
     Expr condition = this->Mutate(op->condition);
     if (condition.type().is_vector()) {
-      LOG(WARNING) << "Detect vector condition in Vectorized Loop, scalarizing...";
```

Comment:
why remove this ?

Reply:
Currently it seems to pollute the logged data; ideally we would just print this once?

Reply:
This message is important, because it tells us when vectorization isn't working because of vectorization axis length vs input shape mismatch.

Reply:
OK, I think we should keep this warning.

Reply:
It does not pollute logging data. It occurs when I use "llvm" as target to build resnet-18.

https://github.com/dmlc/tvm/blob/f33fd5c03d8a2b3972e3b69a79a89d0c9754cd9e/topi/python/topi/x86/conv2d.py#L214
@masahi Can I fix it by checking the length of `w` and only vectorize it when the length of `w` is a multiple of 16?

Reply:
@merrymercy Sure. I am aware of this issue. Probably 8 is better split factor than 16 for imagenet model.